src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -173,14 +173,14 @@ def load_arguments(self, _):
         c.argument('https_only', help="Redirect all traffic made to an app using HTTP to HTTPS.",
                    arg_type=get_three_state_flag(return_label=True))
         c.argument('force_dns_registration', help="If true, web app hostname is force registered with DNS",
-                   arg_type=get_three_state_flag(return_label=True))
+                   arg_type=get_three_state_flag(return_label=True), deprecate_info=c.deprecate(expiration='2.23.0'))
```

Comment:
is there an alternative parameter to use? If yes, please ad redirect info in your deprecation object.

Reply:
No there is no alternate property